components/PromptRemove.vue

```diff
@@ -145,6 +154,9 @@ export default {
   watch:    {
     showPromptRemove(show) {
       if (show) {
+        if (this.currentRouter?.currentRoute?.name === 'c-cluster-explorer-tools' && this.toRemove[0].type === CATALOG.APP && this.toRemove[0].spec?.chart?.metadata?.annotations[CATALOG_ANNOTATIONS.AUTO_INSTALL]) {
```

Comment:
Just a styling comment, this is a pretty long line and should be split up, would suggest each clause is on a new one

Reply:
I cleaned this up a bit.